src/sections/support-page/hero/hero.module.scss

```diff
@@ -0,0 +1,53 @@
+@import '~@bit/bit.base-ui.layout.breakpoints';
+
+.mainGrid {
+	gap: 20px;
+	align-items: center;
+}
+
+.content {
+	margin: 0 auto 25px auto;
+	@media screen and (min-width: $br-md) {
+		margin: 0 0 65px 0;
+	}
+}
+
+.image {
+	max-width: 100%;
+}
+
+.buttons {
+	display: flex;
+	justify-content: center;
+	
+	@media screen and (min-width: $br-md) {
+		justify-content: flex-start;
+	}
+	.link {
+		width: 100%;
+		@media screen and (min-width: $br-sm) {
+			width: unset;
+		}
+	}
+	.button {
+		height: 38px;
+		width: 100%;
+		box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.1);
+		@media screen and (min-width: $br-sm) {
+			width: 180px;
+		}
+	}
+}
+
+.spacer {
+	width: 24px;
+	min-width: 24px;
```

Comment:
why `min-width`

Reply:
this was because the `.buttons` class uses flex and I didnt want the buttons to move closer (because they have `width: 100%`) so it changed the spacer width. but I replaced it with `flex-shrink` so now it is better